modules/Bio/EnsEMBL/Compara/Utils/DistanceMatrix.pm

```diff
@@ -253,14 +243,14 @@ sub collapse_group_in_matrix {
 	my ($matrix, $to_collapse_full, $new_group_id) = @_;
 
 	# print "!!!! collapsing [" . join(',', map {$_->name . '(' . $_->dbID . ')'} @$to_collapse) . "] to $new_group_id\n";
-
```

Comment:
Put back deleted line (and optionally uncomment print to a `if $self->$debug` - since this is out of the scope of changes you were making) , or remove commented line too :+1: